engine/apps/slack/scenarios/resolution_note.py

```diff
@@ -82,10 +83,11 @@ def process_scenario(
                 _slack_team_identity=slack_team_identity,
                 channel_id=channel_id,
             )
-        except KeyError:
-            self.open_warning_window(payload, warning_text)
-            return
-        except SlackMessage.DoesNotExist:
+        except (KeyError, SlackMessage.DoesNotExist):
+            if settings.FEATURE_MULTIREGION_ENABLED:
```

Comment:
I propose to check UNIFIED_SLACK_APP_ENABLED flag. MULTIREGION is already enabled.
P.S.
Yes, these flags need some pruning :)

Reply:
done in https://github.com/grafana/oncall/commit/853a135ccc8e70dcb111e519be3d66ae0543216a